Dockerfile.rhel

```diff
@@ -32,9 +32,8 @@ RUN make build
 #--------------------------------------------------------------------
 
 # On CI , use
-# FROM registry.svc.ci.openshift.org/ocp/ubi-minimal:7
 
-FROM registry.access.redhat.com/ubi7/ubi-minimal
+FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
why not `registry.access.redhat.com/ubi8/ubi-minimal`

Reply:
done https://github.com/redhat-developer/service-binding-operator/pull/316/commits/4c4b2a8ddc0919ec5b401c8b3593e8d91417a15c